en/references/tested-judge/template-description/README.md

```diff
@@ -0,0 +1,392 @@
+---
+title: "TESTed description templates"
+description: "TESTed description templates"
+---
+
+# Documentation description templates
```

Comment:
Is this a reference or a guide? The introduction reads more like a tutorial, yet the file resides under `references`.